### ISSUE TITLE
rebinding cursor keys to seek_* doesn't work with cursor keys when other actions are bound to them as well

### ISSUE THREAD

**Issue by etosan** [arybczak/ncmpcpp#139]
Recently I merged my music libraries from multiple computers into one bigger system.

This made me use ncmpcpp embedded browser much more then before.

Finally I decided to look at ncmpcpp binding system.
As I am both heavy cursor keys and ranger fm user, I rebound 'left', 'right' to browser 'panel' actions (hopefully correctly) like this:
```
def_key "right"
  require_screen "browser"
  require_runnable "enter_directory"
  enter_directory

def_key "left"
  require_screen "browser"
  require_runnable "jump_to_parent_directory"
  jump_to_parent_directory
```

This works as intended! 

But I was always also quite unhappy with defaulting binding for 'left'/'right' as volume controls and I would like to use them as seek_* keys instead. So in addition to previous one I did:
```
def_key "left"
  require_screen "playlist"
  seek_backward

def_key "right"
  require_screen "playlist"
  seek_forward
```

However this doesn't work! 

Instead of seeking, I get something like 'seek attempt + then stuttering'. 

While key is held down, playback moves back a 'frame', but then somehow 'rewinds' and this repeats, in both forward and backward directions.

After multiple config combination attempts, I discovered single way to make "arrow" keys seek, is to remove **all** bindings from bindings file, and use this:
```
def_key "left"
  seek_backward

def_key "right"
  seek_forward
```
Then, however, I lose ability to use cursor as "ranger" keys in file browser.

Why is this happening? 

According to text embedded in sample bindings, ncmpcpp should attempt to seek anytime key is pressed, and even with "require_screen" prefixed, why does it break?

**Comment by arybczak** [comment]
Fixed by a0b662886555be36037222e541e84c2dc5787658.

**Comment by arybczak** [comment]
It was just a limitation of seeking (will be fixed in 0.7.5).

**Comment by etosan** [comment]
Confirmation:

when I compile tree  a0b6628 it seems to work! Thanks for quick fix.
